### PR TITLE
feat: use up2date Chromium user-agents for device descriptors

### LINKED ISSUE #5446: [BUG] Use up to date Chromium version in device User-Agents
Context: https://playwright.slack.com/archives/CSUHZPVLM/p1613146477170900

Actual: Currently if someone will use a Pixel 2, he will get Chrome 75
Expected: He should get an up to date version or the from the current roll

drive-by: maybe add also the latest iPhones.

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes #5446

### CODE REVIEW

**Review by JoelEinbinder — APPROVED** [review]

**Comment by JoelEinbinder** [comment]
Suspicious test failure: `inspector/cli-codegen-java.spec.ts:49:1 › [chromium] should print the correct context options when using a device `

File: packages/common/.npmignore [hunk 1]
```diff
@@ -39,3 +39,5 @@ lib/**/injected/
 !browsers.json
 # Include commit info
 !commitinfo
+# Include generated devices descriptors
+deviceDescriptorsSource.json
```
Comment (JoelEinbinder) [inline packages/common/.npmignore:43]:
```suggestion
!deviceDescriptorsSource.json
```

Comment (mxschmitt) [reply packages/common/.npmignore]: You are faster than the bots report to me! :p

File: tests/inspector/cli-codegen-java.spec.ts [hunk 2]
```diff
@@ -60,13 +60,13 @@ test('should print the correct context options when using a device', async ({ br
...
 
-test('should print the correct context options when using a device and additional options', async ({ browserName, runCLI }) => {
+test('should print the correct context options when using a device and additional options', async ({ browserName, runCLI, playwright }) => {
   test.skip(browserName !== 'webkit');
 
   const cli = runCLI(['--color-scheme=light', '--device=iPhone 11', '--target=java', emptyHTML]);
   const expectedResult = `BrowserContext context = browser.newContext(new Browser.NewContextOptions()
         .setColorScheme(ColorScheme.LIGHT)
-        .setUserAgent("Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0 Mobile/15E148 Safari/604.1")
+        .setUserAgent("${playwright.devices['iPhone 11'].userAgent}")
```
Comment (JoelEinbinder) [inline tests/inspector/cli-codegen-java.spec.ts:69]: This test is no longer testing anything!

Why does it change? Looks like a bug.

File: tests/inspector/cli-codegen-java.spec.ts [hunk 3]
```diff
@@ -46,12 +46,12 @@ test('should print the correct context options for custom settings', async ({ ru
   expect(cli.text()).toContain(expectedResult);
 });
 
-test('should print the correct context options when using a device', async ({ browserName, runCLI }) => {
+test('should print the correct context options when using a device', async ({ browserName, runCLI, playwright }) => {
   test.skip(browserName !== 'chromium');
 
   const cli = runCLI(['--device=Pixel 2', '--target=java', emptyHTML]);
   const expectedResult = `BrowserContext context = browser.newContext(new Browser.NewContextOptions()
-        .setUserAgent("Mozilla/5.0 (Linux; Android 8.0; Pixel 2 Build/OPD3.170816.012) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3765.0 Mobile Safari/537.36")
```
Comment (JoelEinbinder) [inline tests/inspector/cli-codegen-java.spec.ts:54]: leave it as is and just bump the chrome version please.

**Review by JoelEinbinder — APPROVED** [review]

**Pull request merged**